src/navigation/auth/screens/CreateAccount.tsx

```diff
@@ -307,6 +323,32 @@ const CreateAccountScreen: React.VFC<CreateAccountScreenProps> = ({
           />
         </AuthRowContainer>
 
+        <AuthRowContainer>
+          <Controller
+            control={control}
+            render={({field}) => (
+              <>
+                <CheckboxControl>
+                  <Checkbox
+                    onPress={() =>
+                      setValue('agreedToMarketingCommunications', !field.value)
+                    }
+                    checked={field.value}
+                  />
+
+                  <CheckboxLabel>
+                    {t(
+                      'I agree to receive marketing communications from BitPay.',
```

Comment:
Shouldn't this be something like `I would like to receive ...` instead of `I agree to receive ...`